chromium_src/chrome/browser/ui/views/permission_bubble/chooser_bubble_ui.cc

```diff
@@ -9,17 +9,34 @@
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/tabs/tab_strip_model.h"
+#include "chrome/browser/ui/views/title_origin_label.h"
+#include "chrome/grit/generated_resources.h"
+#include "components/grit/brave_components_strings.h"
 #include "content/public/browser/web_contents.h"
+#include "ui/base/l10n/l10n_util.h"
 #include "ui/views/bubble/bubble_dialog_delegate_view.h"
 #include "ui/views/focus/focus_manager.h"
 #include "ui/views/widget/widget.h"
 #include "url/gurl.h"
 
 namespace {
 
-bool IsBravePanel(content::WebContents* content) {
-  return content->GetVisibleURL().EqualsIgnoringRef(
-      GURL(kBraveUIWalletPanelURL));
+bool IsBraveWalletPanel(content::WebContents* content) {
+  return content && content->GetVisibleURL().EqualsIgnoringRef(
+                        GURL(kBraveUIWalletPanelURL));
+}
+
+bool IsBraveWalletPage(content::WebContents* content) {
+  return content && content->GetVisibleURL().EqualsIgnoringRef(
+                        GURL(kBraveUIWalletPageURL));
+}
+
+std::unique_ptr<views::Label> BraveCreateTitleOriginLabel(
+    const std::u16string& source_text) {
```

Comment:
`source_text` is not used.

Reply:
it was actually unused but I removed that patches